docs/release-notes/release-notes-0.20.0.md

```diff
@@ -26,8 +26,22 @@
 
 ## RPC Additions
 
+* [The `lncli addinvoice --blind` command now has the option to include a
+  chained channels incoming list `--blinded_path_incoming_channel_list` which
+  gives users the control of specifying the channels they prefer to receive the 
+  payment on. With the option to specify multiple channels this control can be 
+  extended to multiple hops leading to the
+  node.](https://github.com/lightningnetwork/lnd/pull/9127)
+
 ## lncli Additions
 
+* [The `lncli addinvoice --blind` command now has the option to include a
+  chained channels incoming list `--blinded_path_incoming_channel_list` which
```

Comment:
remove this entry here not need for a double entry, just keep it in `RPC Additions`